src/ripple/protocol/TER.h

```diff
@@ -96,6 +96,7 @@ enum TEMcodes : TERUnderlyingType
     temBAD_SIGNATURE,
     temBAD_SRC_ACCOUNT,
     temBAD_TRANSFER_RATE,
+    temCANT_USE_MASTER_KEY,
```

Comment:
I'm not super thrilled with this name. The existing `temBAD_SIGNER` may be perfectly appropriate, and if not I suggest something like `temBAD_REGKEY`.